tests/model/test_video.py

```
@@ -91,6 +91,10 @@ def test_video_open_close(centered_pair_low_quality_path):
     video.open(grayscale=True)
     assert video.shape == (1100, 384, 384, 1)
 
+    video.open(centered_pair_frame_paths)
+    assert video.shape == (3, 384, 384, 1)
+    assert type(video.backend) == ImageVideo
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)